src/Avalonia.Themes.Default/ComboBox.xaml

```diff
@@ -16,6 +16,13 @@
                             Margin="{TemplateBinding Padding}"
                             HorizontalAlignment="Left"
                             VerticalAlignment="Center"/>
+            <TextBlock
+                            Name="emptyPlaceholder"
+                            Margin="{TemplateBinding Padding}"
+                            HorizontalAlignment="Left"
+                            VerticalAlignment="Center"
+                            Opacity="0"
```

Comment:
`IsVisible` did not worked, so I had to use Opacity instead.

Reply:
```
<Style Selector="ComboBox[SelectedIndex=-1] /template/ ContentControl">
    <Setter Property="Content">
         <Setter.Value>
             <TextBlock Text="" />
         </Setter.Value>
</Style>
```

Reply:
This looks much better but it throws on application start: `System.InvalidOperationException: 'Cannot use a control as a Setter value. Wrap the control in a <Template>.'`

Reply:
Changing it by wrappping the control into a template like this:
```
<Style Selector="ComboBox[SelectedIndex=-1] /template/ ContentControl">
    <Setter Property="Content">
        <Setter.Value>
            <Template>
                <TextBlock Text="" />
            </Template>
        </Setter.Value>
    </Setter>
</Style>
```
doesn' work as the height is still small. And using a `DataTemplate` instead of Template makes the height correct but showing the control as content, not the empty string.